packages/geo/src/lib/offline/geoDB/configFileToGeoDB.service.ts

```diff
@@ -69,6 +70,19 @@ export class ConfigFileToGeoDBService {
                               responseType = 'arraybuffer';
                             }
                             return this.http.get(url, { responseType }).pipe(
+                              catchError((error: HttpErrorResponse) => {
+                                this.messageService.remove(
+                                  downloadMessage.toastId
+                                );
+                                this.messageService.success(
+                                  'igo.geo.indexedDb.data-download-failed',
+                                  undefined,
+                                  { timeOut: 40000 }
```

Comment:
It's a subjective subject but I find the deadline a little long. I usually configure this type of notification to a maximum of 5 seconds otherwise I use a dialog to communicate information that requires special attention